openhands-sdk/openhands/sdk/conversation/impl/remote_conversation.py

```diff
@@ -461,6 +464,13 @@ def __init__(
         self.workspace = workspace
         self._client = workspace.client
 
+        # Build map of log directories for all LLMs in the agent
+        self._log_completion_folders = {}
+        for llm in agent.get_all_llms():
+            if llm.log_completions:
+                # Map usage_id to log folder
+                self._log_completion_folders[llm.usage_id] = llm.log_completions_folder
```

Comment:
I don't think we necessarily need this since we can always get this when needed?